logback/src/main/java/com/linecorp/armeria/common/logback/RequestContextExportingAppender.java

```diff
@@ -144,7 +145,7 @@ public void setExport(String mdcKey) {
      */
     public void setExports(String mdcKeys) {
         requireNonNull(mdcKeys, "mdcKeys");
```

Comment:
Not related with this PR, could you add a `checkArguments` that checkes`mdfKeys` is non-empty?

Reply:
Sure. I added `checkArguments`.